sdk/program/src/message/versions/mod.rs

```diff
@@ -132,6 +132,15 @@ impl VersionedMessage {
         }
     }
 
+    /// Program instructions that will be executed in sequence and committed in
+    /// one atomic transaction if all succeed.
+    pub fn instructions_mut(&mut self) -> &mut [CompiledInstruction] {
```

Comment:
Do you foresee additional use-cases for this beyond that one test? If not, maybe add a note that this is currently only used for tests?

Reply:
Not really, I just reverted this and refactored the test to not need it.